2023-08-11 16:28:10,808 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:28:19,842 ERROR: Exception on /send_message/RunnaRock [POST] [in /Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/danielweir/highlyfe3/venv/lib/python3.9/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
  File "/Users/danielweir/highlyfe3/app/main/routes.py", line 366, in send_message
    if referrer_url.endswith('/user/' + recipient.username):
AttributeError: 'str' object has no attribute 'username'
2023-08-11 16:29:04,228 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:30:25,544 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:31:54,952 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:33:47,110 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:36:45,580 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:40:08,622 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:41:04,797 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:57:44,934 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 16:59:22,517 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
2023-08-11 17:00:03,822 INFO: Highlyfe startup [in /Users/danielweir/highlyfe3/app/__init__.py:59]
